### ISSUE TITLE
[error] HashTable has different value for same key

### ISSUE THREAD

**Issue by denisb411** [tensorflow/nmt#232]
I'm trying to execute the first example:

mkdir /tmp/nmt_model
python -m nmt.nmt \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu

And it's returning the following error:

FailedPreconditionError (see above for traceback): HashTable has different value for same key. Key <unk> has 0 and trying to add value 86

![image](https://user-images.githubusercontent.com/13735079/34577176-6ca18a98-f167-11e7-8444-343c896e512e.png)

How can I solve this?

**Comment by oahziur** [comment]
Hi @denisb411 

Did you follow the instruction to download /tmp/nmt_data/...? Looks like there are duplicated values in your vocab file.

**Comment by newlts** [comment]
I encountered the same error, I solved it by text edior, found and deleted the duplicated word. It works.

**Comment by denisb411** [comment]
Thanks, I will look around on this issue of duplicated words.

**Comment by lmthang** [comment]
Seems like this has been resolved. Close for now.
